BrainPortal/db/migrate/20170220152128_signups_tweaks.rb

```diff
@@ -0,0 +1,14 @@
+class SignupsTweaks < ActiveRecord::Migration
```

Comment:
This migration should fill the new user_id column for all the records already present and approved, the best it can, using the old logic of matching login names.